### DISCUSSION TITLE
Checking the schedule?

### DISCUSSION THREAD

**Discussion by brandoncc** [sidekiq/sidekiq#5013] [Q&A] [answered]
Is it possible to:

1. Check if there are any items in the schedule
2. Check if a job with a specific class and set of arguments are in the schedule

I'm totally fine if there is a race condition. If it was correct 95% of the time I'd be satisfied. 

Thanks!

**Comment by mperham** [comment, marked as answer]
Documented here: https://github.com/mperham/sidekiq/wiki/API#scheduled

**Reply by brandoncc** [reply]
Thank you, Mike! I looked all over the place in the tests and code...but I didn't remember that wiki page.
